cves/kernel/CVE-2021-3178.yml

```diff
@@ -147,10 +148,12 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    This vulnerability was discovered and reported by a user with no apparent
+    affiliation with any organization or event.
```

Comment:
I'm assuming this was derived from the `reported-by` field in the fix commit https://github.com/torvalds/linux/commit/51b2ee7d006a736a9126e8111d1f24e4fd0afaa6 ?